config/task-monitor-result.yaml

```diff
@@ -0,0 +1,71 @@
+# Monitor the pipelinerun status and update the pull request
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: monitor-result-task
+spec:
+  inputs:
+    resources:
+      - name: pull-request
+        type: pullRequest
+    params:
+      - name: pipelinerun
+        description: The name of the pipelinerun to be monitored
+        default: default pipelinerun
+      - name: commentsuccess
+        description: The text of the sccess comment
+        default: default success at task
+      - name: commentfailure
+        description: The text of the failure comment
+        default: default failure at task
+  outputs:
+    resources:
+      - name: pull-request
+        type: pullRequest
+  steps:
+  - name: check
+    image: ibmcom/microclimate-utils:1905
```

Comment:
why use this image and not any other with bash in that we don't have to maintain?

Reply:
I wanted the image with kubectl but I can change to use the python kubernetes client library and use other generic image.

Reply:
I switched to the image with the python kubernetes client.